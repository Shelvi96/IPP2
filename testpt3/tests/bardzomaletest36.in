NEW ID

:>58;
8;:46>5094
4;:30923;;>2374
007086;14>:
123:155>27166
8>50955451
86:14627:>079624782
84:0135:>;415322991
11738973>:95679;3
9822>;945
56::0>1861333489
5870:76>8:143;929
265>0653123:6
;51148>06:91963;:
@4;::91435696527667
@:517736264430713
@777418:;295051;:6:99843587530
@0737;9818
@;5;862:1082
@06285;358353340;
@8996850:353;8
@3
@63;5785:191:1082258::0156
@560006433;;915:37;731
@010:
