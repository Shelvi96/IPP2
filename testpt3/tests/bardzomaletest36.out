100210
730
186498079633391616
0
0
8
1
0
3164580176175
100200010
0
